### DISCUSSION TITLE
feat: Image Transformation.

### DISCUSSION THREAD

**Discussion by jamaluddinrumi** [SonicJs-Org/sonicjs#242] [General]
it's a wonderful project. i agree with cloudflare is the fastest platform in performance.

but, how can we provide multiple generated responsive images served by R2 for our img srcset?

similar feature is provided by [storyblok image](https://www.storyblok.com/tp/optimize-your-storyblok-images-with-image-engine)

**Comment by lane711** [comment]
We don't currently have image transformation, but I will add it to the backlog. Thanks for asking

**Comment by lane711** [comment]
here is the issue for the backlog:
https://github.com/lane711/sonicjs/issues/243
